src/searching.md

```diff
@@ -183,148 +182,151 @@ Searches that ignore combining characters are slower than regular searches.
 Anki 2.1.24+, AnkiMobile 2.0.60+ and AnkiDroid 2.17+ support searching in notes with "regular expressions",
 a standard and powerful way of searching in text.
 
-Start a search with `re:` to search by regular expression. To make things easier, Anki will
+Start a search with `re:` to search using regular expression. To make things easier, Anki will
 treat the following as [raw input](#raw-input), so bear in mind the rules listed there.
 
 Some examples:
 
 `"re:(some|another).*thing"`\
-find notes that have "some" or "another" on them, followed by 0 or more characters, and then "thing"
+finds notes that have "some" or "another" on them, followed by 0 or more characters, and then "thing".
 
 `re:\d{3}`\
-find notes that have 3 digits in a row
+finds notes that have 3 digits in a row.
 
 Regular expressions can also be limited to a specific field. Please note that unlike the normal searches
-in a specific field, regular expressions in fields don't require an exact match. Eg:
+in a specific field, regular expressions in fields don't require an exact match:
 
 `front:re:[a-c]1`\
-matches uppercase or lowercase a1, B1 or c1 that occurs anywhere in the "Front" field
+matches uppercase or lowercase a1, B1 or c1 that occurs anywhere in the "Front" field.
 
 `front:re:^[a-c]1$`\
-like the above, but will not match if any other text falls before or after a1/b1/c1.
+same as previous, but will not match if any other text falls before or after a1/b1/c1.
 
-Anki 2.1.50 added regex support for tags:
+Anki 2.1.50+ supports regular expressions for tags:
```

Comment:
Are there AnkiMobile and AnkiDroid versions that could be added here?

Reply:
AnkiDroid 2.17 probably, I'm not sure about AnkiMobile. @dae?

Reply:
I'm not sure this is exposed by AnkiMobile yet.